crates/iota-light-client/example_config/light_client.yaml

```diff
@@ -1,3 +1,6 @@
-full_node_url: "http://0.0.0.0:9000"
-checkpoint_summary_dir: "example_config"
+full_node_url: "http://localhost:9000"
+# Provide an existing directory that contains the genesis file of the network
+checkpoint_summary_dir: ""
 genesis_filename: "genesis.blob"
+object_store_url: ~
```

Comment:
maybe add an entry in your issue to change that in the future?

Reply:
Did you add it?

Reply:
I am pretty sure I added it, but it didn't publish it for some reason.

Reply:
I was thinking of https://github.com/iotaledger/iota/issues/6090 but all good